src/image/mod.rs

```diff
@@ -11,31 +11,38 @@
 //!
 //! # Examples
 //!
-//! ## Load a TGA image and draw it to a display
+//! ## Load an RGB565 raw data imge and display it
 //!
-//! This example loads a TGA-formatted image using the [tinytga] crate and draws it to the display
-//! using an [`Image`] object.
+//! This example loads a small image from a raw data array and displays it. The image is RGB565
+//! encoded, so uses the `Rgb565` color type.
 //!
 //! The `graphics` feature of `tinytga` needs to be enabled in `Cargo.toml` to use the `Tga` object
 //! with embedded-graphics.
 //!
-//! ```rust,ignore
-//! use embedded_graphics::{image::Image, pixelcolor::Rgb888, prelude::*};
+//! ```rust
+//! use embedded_graphics::{
+//!     image::{Image, ImageRaw, ImageRawBE},
+//!     pixelcolor::Rgb565,
+//!     prelude::*,
+//! };
 //! # use embedded_graphics::mock_display::MockDisplay as Display;
-//! use tinytga::Tga;
 //!
-//! let mut display: Display<Rgb888> = Display::default();
+//! let mut display: Display<Rgb565> = Display::default();
 //!
-//! // Load the TGA file.
-//! // Note that the color type is set explicitly to match the format used in the TGA file,
+//! // Raw image data for demonstration purposes.
+//! let data = [
+//!     0x00, 0x00, 0xF8, 0x00, 0x07, 0xE0, 0xFF, 0xE0, //
+//!     0x00, 0x1F, 0x07, 0xFF, 0xF8, 0x1F, 0xFF, 0xFF, /*
+//!            * ... */
+//! ];
```

Comment:
The data array can't be any other size for the given image size. But we could add a comment that  real image data would likely be longer.
```suggestion
//! let data = [
//!     0x00, 0x00, 0xF8, 0x00, 0x07, 0xE0, 0xFF, 0xE0, //
//!     0x00, 0x1F, 0x07, 0xFF, 0xF8, 0x1F, 0xFF, 0xFF, //
//! ];
```